pkg/cri/sbserver/image_pull.go

```diff
@@ -318,7 +318,7 @@ func (c *criService) getTLSConfig(registryTLSConfig criconfig.TLSConfig) (*tls.C
 		if len(cert.Certificate) != 0 {
 			tlsConfig.Certificates = []tls.Certificate{cert}
 		}
-		tlsConfig.BuildNameToCertificate() // nolint:staticcheck
+		tlsConfig.BuildNameToCertificate() //nolint:staticcheck
```

Comment:
same as https://github.com/containerd/containerd/pull/7349/files#r990644833 (this code was forked from `pkg/cri/server`